pkg/infrastructure/clusterapi/clusterapi.go

```diff
@@ -52,9 +52,6 @@ const (
 	ignitionStage            = "Bootstrap Ignition Provisioning"
 	machineStage             = "Machine Provisioning"
 	postProvisionStage       = "Infrastructure Post-provisioning"
-
-	// CAPIArtifactsDir is the directory where the manifests generated by CAPI are stored.
-	CAPIArtifactsDir = ".clusterapi_output"
```

Comment:
I probably should have explained this in the commit message. This PR moves this to pkg/clusterapi to avoid circular dependencies. It seems like a good fit.